be/src/storage/persistent_index.cpp

```diff
@@ -847,8 +847,8 @@ size_t MutableIndex::estimate_nbucket(size_t key_size, size_t size, size_t nshar
     return std::min(kBucketPerPage, npad(size, pad));
 }
 
-struct StringHash {
-    size_t operator()(const std::string& s) const { return key_index_hash(s.data(), s.length() - kIndexValueSize); }
+struct StringHasher2 {
```

Comment:
Why not use call key_index_hash directly

Reply:
it called in several places, and also need pass it to phmap as a callable object.